### PR TITLE
Filter sort and pin feature

### PR DESCRIPTION
**Pull request by mansipadgelwar**
Filter, sort, and Pin notes feature.
- Filter by tags
     I can filter notes by tags added to the note.
- Filter by priority
    I can filter the notes by priority
    Priorities would be high, medium, low
- Sort by date
   I can sort the notes by the date added/created.

### CODE REVIEW

**Review by ravikumar1002 — APPROVED** [review]
function working good
good to merge

File: src/pages/Home/Home.jsx [hunk 1]
```diff
@@ -87,42 +114,95 @@ const Home = () => {
...
+                              : "material-icons-outlined"
+                          }
+                        >
+                          push_pin
+                        </span>
+                      </div>
+                    </div>
+                    <div
+                      className="notes-body"
+                      dangerouslySetInnerHTML={{ __html: item.body }}
```
Comment (Developers-codz) [inline src/pages/Home/Home.jsx:141]: this is new to me what does this attribute do?

**Review by Developers-codz — APPROVED** [review]
Good work on modularising CSS, code quality is also good found nothing to suggest 
can go for merge!

File: src/pages/Home/Home.jsx (hunk 1 above)
Comment (mansipadgelwar) [reply src/pages/Home/Home.jsx:141]: it sets HTML in react. similar to innerHTML.

**Pull request merged**